cpp/CMakeLists.txt

```diff
@@ -226,6 +230,17 @@ if(ARROW_BUILD_TESTS OR ARROW_BUILD_BENCHMARKS)
   set(ARROW_WITH_ZSTD ON)
 endif()
 
+if (WIN32)
+  # ORC doesn't build on windows
```

Comment:
Are there JIRAs open about this on the Apache ORC project?

Reply:
https://issues.apache.org/jira/browse/ORC-34, but it's also an issue with the cmake build system assuming *nix in some places (e.g. calling `./configure` directly).